src/sentry/workflow_engine/migration_helpers/rule.py

```diff
@@ -0,0 +1,112 @@
+from typing import Any
+
+from sentry.issues.grouptype import ErrorGroupType
+from sentry.models.organization import Organization
+from sentry.models.rule import Rule
+from sentry.rules.processing.processor import split_conditions_and_filters
+from sentry.users.services.user import RpcUser
+from sentry.workflow_engine.migration_helpers.issue_alert_conditions import (
+    translate_to_data_condition,
+)
+from sentry.workflow_engine.models import (
+    AlertRuleDetector,
+    AlertRuleWorkflow,
+    DataConditionGroup,
+    Detector,
+    DetectorWorkflow,
+    Workflow,
+    WorkflowDataConditionGroup,
+)
+
+
+def migrate_issue_alert(rule: Rule, user: RpcUser | None = None):
+    data = rule.data
+    project = rule.project
+    organization = project.organization
+
+    error_detector, _ = Detector.objects.get_or_create(
+        type=ErrorGroupType.slug, project=project, defaults={"config": {}, "name": "Error Detector"}
+    )
+    AlertRuleDetector.objects.create(detector=error_detector, rule=rule)
+
+    conditions, filters = split_conditions_and_filters(data["conditions"])
+    when_dcg = create_when_dcg(
```

Comment:
nit: `workflow_when_dcg` to help illustrate it's unrelated to detectors / grouping rules.